docs/syntax_and_semantics/offsetof.md

```diff
@@ -1,9 +1,19 @@
 # offsetof
 
-An `offsetof` expression returns the byte offset of an instance variable in a struct or class type. It accepts the type as first argument and the instance variable name prefixed by an `@` as second argument:
+An `offsetof` expression returns the byte offset of a subobject in an instance of a class or struct.
```

Comment:
Not sure if `subobject` is an approriate term here. Maybe `property` is better?

Reply:
"Instance property" doesn't carry the same connotation here because a "property" needn't be tied to an instance's memory representation. Maybe "instance member" or "instance field" is better.

Reply:
I suppose `field` is a common term for this.